### PR TITLE
Feat/create main page

### PR DESCRIPTION
**Pull request by vlzelensky**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import React from 'react';
+import { Header, Books } from 'Components';
 
 function App() {
```
Comment (dariausatova) [inline src/App.tsx]: es6 function

File: src/Components/Books/Books.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { books } from 'store';
+import { Book } from './modules';
+import { useLazyLoad } from 'hooks';
+
+import './styles.css';
+
+export const Books = observer(() => {
+  const { data, onScroll } = useLazyLoad(books.books, 9);
```
Comment (dariausatova) [inline src/Components/Books/Books.tsx]: can you please avoid duplication of names?

File: src/Components/Books/modules/Book/Book.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import '../../styles.css';
+
+interface BookProps {
```
Comment (dariausatova) [inline src/Components/Books/modules/Book/Book.tsx]: please put the interfaces in a separate file or files

File: src/Components/Books/modules/Book/Book.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  id: number;
+  author: string;
+  name: string;
+  image: string;
+  isTaken: boolean;
+}
+
+export const Book: FC<BookProps> = ({ id, author, name, image, isTaken }) => (
+  <div className={`book ${isTaken ? 'book_isTaken' : ''}`} key={id}>
```
Comment (dariausatova) [inline src/Components/Books/modules/Book/Book.tsx]: use && in ternary operator

File: src/Components/Books/modules/Book/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export * from './Book';
```
Comment (dariausatova) [inline src/Components/Books/modules/Book/index.ts]: Снимок экрана 2022-08-15 в 14 43 38
why are there 2 identical imports?

File: src/Components/Books/styles.css [hunk 6]
```diff
@@ -0,0 +1,53 @@
+.container {
+    width: 100%;
```
Comment (dariausatova) [inline src/Components/Books/styles.css]: why are there 4 indents?

File: src/store/books.ts [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import { makeAutoObservable } from 'mobx';
+import { booksData } from 'consts';
+import { BookI } from 'types';
+
+class Books {
+  constructor() {
+    makeAutoObservable(this);
+  }
+  books: BookI[] = booksData;
```
Comment (dariausatova) [inline src/store/books.ts:9]: Снимок экрана 2022-08-15 в 14 48 25
why is there one with : and one with assignment?

File: src/Components/Books/modules/Book/index.ts (hunk 5 above)
Comment (vlzelensky) [reply src/Components/Books/modules/Book/index.ts]: first one exports from file, the second one exports from directory

**Pull request merged**